content/23.vaccines-novel.md

```diff
@@ -192,7 +192,7 @@ The phase 2 results supported that the vaccine was safe, with 1,446 treatment-re
 The single grade 3 event was joint stiffness [@doi:10.1101/2021.05.07.21256652].
 The rates of AEs in the placebo group are not reported.
 In terms of immunogenicity, pre- and post-vaccination blood samples were again collected and were evaluated for a humoral immune response to the spike protein, and the treatment group was identified to show significantly greater neutralizing activity than the placebo group [@doi:10.1101/2021.05.07.21256652].
-The phase 2/3 trials are ongoing in several countries, including the United States, Mexico, India, and Colombia [@url:https://ir.inovio.com/news-releases/news-releases-details/2021/INOVIO-Receives-U.S.-FDA-Authorization-to-Proceed-with-INNOVATE-Phase-3-Segment-for-its-COVID-19-Vaccine-Candidate-INO-4800-in-the-U.S/default.aspx; @url:https://ir.inovio.com/news-releases/news-releases-details/2021/INOVIO-Further-Expands-INNOVATE-Phase-3-Trial-for-COVID-19-DNA-Vaccine-Candidate-INO-4800-With-Regulatory-Authorization-from-India/default.aspx; @url:https://ir.inovio.com/news-releases/news-releases-details/2021/INOVIO-Expands-INNOVATE-Phase-3-for-INO-4800-its-DNA-Vaccine-Candidate-for-COVID-19-to-include-Colombia-following-Regulatory-Authorization/default.aspx; @url:https://ir.inovio.com/news-releases/news-releases-details/2021/INOVIO-Receives-Regulatory-Authorization-to-Conduct-Phase-3-Efficacy-Trial-of-its-COVID-19-DNA-Vaccine-Candidate-INO-4800-in-Mexico/default.aspx].
+The phase 2/3 trials are ongoing in several countries, including the United States, Mexico, India, and Colombia [@{url:https://ir.inovio.com/news-releases/news-releases-details/2021/INOVIO-Receives-U.S.-FDA-Authorization-to-Proceed-with-INNOVATE-Phase-3-Segment-for-its-COVID-19-Vaccine-Candidate-INO-4800-in-the-U.S/default.aspx}; @{url:https://ir.inovio.com/news-releases/news-releases-details/2021/INOVIO-Further-Expands-INNOVATE-Phase-3-Trial-for-COVID-19-DNA-Vaccine-Candidate-INO-4800-With-Regulatory-Authorization-from-India/default.aspx}; @{url:https://ir.inovio.com/news-releases/news-releases-details/2021/INOVIO-Expands-INNOVATE-Phase-3-for-INO-4800-its-DNA-Vaccine-Candidate-for-COVID-19-to-include-Colombia-following-Regulatory-Authorization/default.aspx}; @{url:https://ir.inovio.com/news-releases/news-releases-details/2021/INOVIO-Receives-Regulatory-Authorization-to-Conduct-Phase-3-Efficacy-Trial-of-its-COVID-19-DNA-Vaccine-Candidate-INO-4800-in-Mexico/default.aspx}].
```

Comment:
Would skip these (if they start with `{`, I assume human eyes have chosen a way to resolve some issue that is beyond my script)

Reply:
Yes, I noticed in the build log that the punctuation was causing a prefix of the URL be to used instead of the entire URL.  Wrapping in `{ }` fixed that.